modules/quantcastBidAdapter.js

```diff
@@ -18,6 +19,9 @@ export const QUANTCAST_TEST_PUBLISHER = 'test-publisher';
 export const QUANTCAST_TTL = 4;
 export const QUANTCAST_PROTOCOL = 'https';
 export const QUANTCAST_PORT = '8443';
+export const QUANTCAST_FPA = '__qca';
+
+export const storage = getStorageManager();
```

Comment:
Does Quantcast have a registered GVL id?

Passing the following will allow storage access when GDPR Enforcement is enabled.
https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L61

Reply:
We do. I updated the code accordingly. Thanks for the catch.